packages-exp/firebase-exp/package.json

```diff
@@ -25,107 +25,185 @@
   ],
   "exports": {
     "./analytics": {
-      "node": "./analytics/dist/index.cjs.js",
+      "node": {
+        "require": "./analytics/dist/index.cjs.js",
+        "import": "./analytics/dist/index.esm.js"
```

Comment:
The file extension should be `.mjs`, otherwise Node would fail to parse it.

Reply:
Indeed, good catch! I fixed it in a different way by declaring `"type": "module"` in the `package.json`s, and changing the extension of cjs files from js to cjs, so they still work with `require()`s. This way, I only need to generate 2 files per subpath, an esm.js file and a cjs file instead of 3(esm.js, esm.mjs and cjs.js).

Reply:
I tested it locally, and both import and require syntax work with this setup.